app/components/work-item/WorkItemHeader.vue

```
@@ -123,6 +124,84 @@ const ciSummary = computed(() => {
+})
+
+const STRATEGY_VISUALS = {
+  merge: ['i-lucide-circle', 'i-lucide-circle', 'i-lucide-circle', 'i-lucide-git-merge', 'i-lucide-arrow-right'],
+  squash: ['i-lucide-circle', 'i-lucide-circle', 'i-lucide-circle', 'i-lucide-chevrons-right', 'i-lucide-git-commit-horizontal'],
+  rebase: ['i-lucide-arrow-right', 'i-lucide-circle', 'i-lucide-circle', 'i-lucide-circle', 'i-lucide-arrow-right'],
+} as const
+
+const mergeStrategies = computed(() => {
+  const allowed = mergeStatus.value?.allowedStrategies ?? ['merge', 'squash', 'rebase']
+  const count = mergeStatus.value?.commitCount ?? 0
+  return allowed.map((value) => {
```

Biome (2.4.4)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)